hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -212,6 +224,44 @@ def _bank_add_signals(
     }
 
 
+@bp.route("/bank/<bank_name>/content/<content_id>", methods=["PUT"])
+def bank_update_content(bank_name: str, content_id: int):
+    """
+    Update the metadata for a banked content item.
+
+    Inputs:
+        * disable_until_ts (optional): Unix timestamp in seconds.
+    """
+    storage = persistence.get_storage()
+    bank = storage.get_bank(bank_name)
+    if not bank:
+        abort(404, f"bank '{bank_name}' not found")
+    content = storage.bank_content_get([content_id])
+    if not content:
+        abort(404, f"content '{content_id}' not found")
+    content = content[0]
+    data = request.get_json()
+
+    try:
+        if "disable_until_ts" in data:
+            disable_until_ts = flask_utils.str_to_type(data["disable_until_ts"], int)
+            if disable_until_ts < 0:
+                raise ValueError("disable_until_ts must be a non-negative integer")
+            if disable_until_ts > MAX_TIME:
+                raise ValueError(
+                    "disable_until_ts must be less than 100 years in the future"
+                )
+            content.disable_until_ts = disable_until_ts
+        storage.bank_content_update(content)
+    except ValueError as e:
+        abort(400, *e.args)
+    except KeyError as e:
+        abort(404, *e.args)
```

Comment:
You are also potentially catching exceptions from `storage.bank_content_update(content)` which may need to be a 500. 

Instead of throwing, catching, and aborting, just `abort(400`) inline.